src/app/beer_garden/log.py

```diff
@@ -111,16 +111,23 @@ def setup_entry_point_logging(queue):
     )
 
 
-def get_plugin_log_config(system_name=None):
-    return _PLUGIN_LOGGING.get_plugin_log_config(system_name=system_name)
+def get_plugin_log_config(**_) -> dict:
+    """Get the plugin logging configuration
 
+    Args:
+        **_: Eventually you will be able to select a plugin logging config based on
+            selectors (system name, etc.)
```

Comment:
When we chatted about this, part of me was thinking what would happen if we added the logging configuration to the System model? Then when a System is created the plugin would pass the flag is a local config is being used or not. If a local config was not being used, then we could grab the default config and populate it.

This would allow us to later be able to create an admin interface that could change the logging configuration. Just kick off an Admin command and then update the model. Then we would have to determine if a server "forced" logging change overwrites locally provided configs and if they would persist on reboot.

This is not an ask for this PR, just wanted to put my thoughts some where.

Reply:
Hmm, that's a really interesting idea. We'd probably need some sort of granularity at the Instance level too, but I like this idea.